.github/workflows/wiki-documentation.yaml

```diff
@@ -0,0 +1,36 @@
+# This workflow publishes Markdown docs to Github Wiki.
+# Some manual setup is required before using it:
+#   1. Enable Wiki in repository and create at least one page.
+#   2. Generate a Personal Access Token (PAT) with "repo" scope enabled.
+#   3. Add the generated PAT as a secret named "GH_PERSONAL_ACCESS_TOKEN".
+# For details, see https://github.com/marketplace/actions/publish-to-github-wiki#setup
+name: Generate Wiki Pages
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  generate-wiki:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v3
```

Comment:
We switched to using hashes to have more accurate version pins, e.g.

https://github.com/jenstroeger/python-package-template/blob/bb7063271508b13be85b86ee3b9970a1dcdf9d76/.github/workflows/check-and-release-main.yaml#L26-L27

Also, we may want to [harden the Runner](https://github.com/step-security/harden-runner) before checking out the repo:

https://github.com/jenstroeger/python-package-template/blob/bb7063271508b13be85b86ee3b9970a1dcdf9d76/.github/workflows/check-and-release-main.yaml#L22-L25